README.md

```diff
@@ -34,13 +34,15 @@ hesitate to read the source code and understand the mechanism of this bot.
 - [x] **Dry-run**: Run the bot without playing money.
 - [x] **Backtesting**: Run a simulation of your buy/sell strategy.
 - [x] **Strategy Optimization by machine learning**: Use machine learning to optimize your buy/sell strategy parameters with real exchange data.
-- [x] **Whitelist crypto-currencies**: Select which crypto-currency you want to trade.
+- [x] **Risk Reward calculation based trading**: Builtin win rate and risk reward ratio using [edge](https://github.com/freqtrade/freqtrade/blob/develop/docs/edge.md).
```

Comment:
Nice !  I have not thought about feature list. 
U would put it as follows though: 
** Edge position sizing ** Calculate your win rate, risk reward ratio and the best stoploss and adjust your position size before taking a position for each specific market.